src/calibration.c

```diff
@@ -147,13 +148,13 @@ void run_movement_sequence(const char *sequence)
 			stop_end();
 			break;
 		case 'l':
-			turn_left();
+			speed_turn(MOVE_LEFT, speed);
```

Comment:
I would include this function on move.c file instead on speed.c and name it turn_side? (to be similar as move_side)

Reply:
Open a new issue if you want. I am willing to re-think the current API, but it is not a trivial change, so we should not do it now I think. :wink:

With the new functions I tried to keep the "module_name" style, so it should start with `speed_` if it is in the `speed.c` module. It it not in  the `move.c` module because now it has a new parameter `speed`. We should probably, instead, move `turn_back` to speed module. Turn back should probably be a `speed_turn()` with 180-degree turn. If there is a front-wall adjustment it should be called `move_back()`. We have a function named like that to get out of a dead-end, so if we want to have a static turn back with front-wall adjustment we should probably call it `turn_back_static()` or something like that.

This is just a first step. There is still a lot of work/refactoring needed. :blush:

Reply:
new issue: #352